arm-documentdb/2015-04-08/documentdb.json

```diff
@@ -0,0 +1,767 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DocumentDB",
+    "description": "Azure DocumentDB Resource Provider REST API",
+    "version": "2015-04-08"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}?api-version={api-version}": {
```

Comment:
There is no need to specify api-version in the path. Path just contains path parameters.
Query parameters are defined in the parameters section with the property `"in": "query"`.